compiler/src/java_plugin/cpp/java_generator.cpp

```diff
@@ -48,7 +107,10 @@ static string MixedLower(const string& word) {
       after_underscore = false;
     }
   }
-  return w;
+  if (java_keywords.find(w) == java_keywords.end()) {
```

Comment:
Could you invert this condition so the final `return` is the common/successful case?